packages/core/src/App/Containers/QuestionnaireModal/questionnaire-modal.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Analytics } from '@deriv/analytics';
+import { Modal } from '@deriv/components';
+import QuestionnaireContent, { TQuestionnaireModal } from 'App/Containers/QuestionnaireModal/questionnaire-content';
+import './questionnaire-modal.scss';
+
+const QuestionnaireModal = ({ ab_questionnaire, onSignupComplete }: TQuestionnaireModal) => {
+    React.useEffect(() => {
+        const modal_root = document.getElementById('modal_root');
+        if (modal_root) modal_root.style.backgroundColor = 'unset';
+        Analytics.trackEvent('ce_questionnaire_form', {
+            action: 'open',
+            question_code: '1',
+            question_content: ab_questionnaire,
+        });
+        return () => {
+            if (modal_root) modal_root.style.backgroundColor = 'rgba(0,0,0,.72)';
```

Comment:
default one for `modal_root` where I need change backgroundColor while quest modal exist

Reply:
is another dialog still appears behind your modal? does the overlay for your modal piles up on top of another modal (dialog)? there should be just 1 modal on the screen at a time :)

Reply:
is another dialog still appears behind your modal? does the overlay for your modal piles up on top of another modal (dialog)? there should be just 1 modal on the screen at a time :) you won't need to change the color in this case

Reply:
discussed, decided to remove Modal at all and integrate in existing flow